INSTALL.rst

```diff
@@ -178,6 +178,18 @@ All variables can be set using ``-DMY_VARIABLE=MY_VARIABLES_VALUE``.
 
 .. website_custom_end
 
+Note on Conda
```

Comment:
Maybe change this to "Note on Building Cyclus with Conda".  I think that specificity helps here.